### PR TITLE
[data-hub] new open research gmail account data pipeline

### PR DESCRIPTION
**Pull request by HazalCiplak**
https://github.com/elifesciences/data-hub-issues/issues/333

### CODE REVIEW

File: deployments/data-hub/data-hub--stg.yaml [hunk 1]
```diff
@@ -193,6 +193,9 @@ spec:
       - name: gmail-secret-volume
```
Comment (de-code) [inline deployments/data-hub/data-hub--stg.yaml:193]: minor: might be worth renaming as well?

**Review by de-code — DISMISSED** [review]

File: deployments/data-hub/data-hub-configs.yaml [hunk 2]
```diff
@@ -47,22 +47,43 @@ data:
   gmail-data-pipeline.config.yaml: |-
     projectName: 'elife-data-pipeline'
     datasetName: '{ENV}'
-    gmailLabelData:
```
Comment (scottaubrey) [inline deployments/data-hub/data-hub-configs.yaml:50]: Just want to check this config file change isn't going to break existing prod image?

File: deployments/data-hub/data-hub--stg.yaml [hunk 3]
```diff
@@ -143,12 +143,12 @@ spec:
           value: "100"
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /dag_secret_files/gcloud/credentials.json
-        - name: GMAIL_ACCOUNT_SECRET_FILE
-          value: /dag_secret_files/gmail/gmail_credentials.json
+        - name: GMAIL_PRODUCTION_ACCOUNT_SECRET_FILE
+          value: /dag_secret_files/gmail_credentials.json
+        - name: GMAIL_OPEN_RESEARCH_ACCOUNT_SECRET_FILE
+          value: /dag_secret_files/gmail_open_research_credentials.json
```
Comment (scottaubrey) [inline deployments/data-hub/data-hub--stg.yaml:149]: Should these two paths not be
- ```/dag_secret_files/gmail/gmail_credentials.json```
- ```/dag_secret_files/gmail_open_research/gmail_open_research_credentials.json```
so they match line 194 and 197 ?

File: deployments/data-hub/data-hub--stg.yaml [hunk 4]
```diff
@@ -190,9 +190,12 @@ spec:
       - name: aws-secret-volume
         mountPath: /home/airflow/.aws
         readOnly: true
-      - name: gmail-secret-volume
+      - name: gmail-production-secret-volume
         mountPath: /dag_secret_files/gmail/
```
Comment (de-code) [inline deployments/data-hub/data-hub--stg.yaml:194]: and rename the path maybe as well?

File: deployments/data-hub/data-hub-configs.yaml (hunk 2 above)
Comment (de-code) [reply deployments/data-hub/data-hub-configs.yaml:50]: Also well spotted.

Comment (HazalCiplak) [reply deployments/data-hub/data-hub-configs.yaml:50]: Gmail pipeline in production is not working today after seeing the staging works after I will make same changes on production too. Is there any other suggestion I can do, maybe changing prod or staging both together?

File: deployments/data-hub/data-hub--stg.yaml (hunk 4 above)
Comment (HazalCiplak) [reply deployments/data-hub/data-hub--stg.yaml:194]: To change the path, I think I should create new secret if I am correct?

File: deployments/data-hub/data-hub-configs.yaml (hunk 2 above)
Comment (HazalCiplak) [reply deployments/data-hub/data-hub-configs.yaml]: Added new version of gmail config file

**Review by de-code — APPROVED** [review]

**Pull request merged**